docs/sql-data-sources-avro.md

```diff
@@ -199,7 +199,7 @@ Data source options of Avro can be set via:
     <td><code>avroSchema</code></td>
     <td>None</td>
     <td>Optional Avro schema provided by a user in JSON format. The date type and naming of record fields
-    should match the input Avro data or Catalyst data, otherwise the read/write action will fail.</td>
+    should match the input Avro data or Spark's internal data type (e.g., StringType, IntegerType)., otherwise the read/write action will fail.</td>
```

Comment:
> should match the input Avro data when reading from Avro source or should match the Spark's internal data type (e.g., StringType, IntegerType) when writing to Avro source; otherwise the read/write action will fail.

How about this?

Reply:
+1 with @gatorsmile , we can also remove the second "should".
```
should match the Avro data type when reading from Avro or match the Spark's internal data type (e.g., StringType, IntegerType) when writing to Avro files; otherwise, the read/write action will fail.
```

Reply:
+1  @gengliangwang reads better. 

"should match the Avro data type when reading from Avro or match the Spark's internal data type (e.g., StringType, IntegerType) when writing to Avro files; otherwise, the read/write action will fail.